app.js

```diff
@@ -305,6 +316,7 @@ function getGridSize() {
  * Sets the grid size value in local storage to a new size.
  * @param {number} size - The new grid size.
 */
+// This could be part of updateGridSize imo
```

Comment:
Since you call this setGridSize only once, I think it would be cleaner inside updateGridSize(). Both functions are small enough. And the problem I ran into was having to track down function call after function call (more on this in final comments)